gui/slick/js/core.js

```diff
@@ -3176,6 +3178,98 @@ var SICKRAGE = {
                     }
                 });
             };
+            
+            /*
+             * Add's shows by by indexer and indexer_id with a number of optional parameters
+             * The show can be added as an anime show, by providing the data attribute: data-isanime="1"
+             */
+            $.initAddShowById = function(){
+                
+                $(document.body).on('click', 'a[data-add-show]', function(e){
+                    e.preventDefault();
+                    
+                    var url = $(this).attr('href');
+                    // Whe're going to add this show, let's remove the anchor and button desc, so it can't be added twice!
+                    if ( $(this).attr('disabled') === 'disabled' ) { return false; }
+
+                    $(this).html('Being Added').attr('disabled', 'disabled');
+                    
+                    var anyQualArray = [];
+                    var bestQualArray = [];
+                    $('#anyQualities option:selected').each(function (i, d) {
+                        anyQualArray.push($(d).val());
+                    });
+                    $('#bestQualities option:selected').each(function (i, d) {
+                        bestQualArray.push($(d).val());
+                    });
+                    
+                    // If we are going to add an anime, let's by default configure it as one
+                    var anime = $('#anime').prop('checked');
+                    var configureShowOptions = $('#configure_show_options').prop('checked');
+                    
+                    /* Let's disable this for now, generates more questions then it saves the user time
+                    if ( !configureShowOptions && $(this).data("isanime") ) {
+                        anime = true;
+                        configureShowOptions = true;
+                    }*/
+
+                    $.get(url + '?indexer_id=' + $(this).attr('data-indexer-id'), {
+                        'root_dir': $('#rootDirs option:selected').val(),
+                        'configure_show_options': configureShowOptions,
+                        'indexer': $(this).attr('data-indexer'),
+                        'show_name': $(this).attr('data-show-name'),
+                        'quality_preset': $('#qualityPreset').val(),
+                        'default_status': $('#statusSelect').val(),
+                        'any_qualities': anyQualArray.join(','),
+                        'best_qualities': bestQualArray.join(','),
+                        'default_flatten_folders': $('#flatten_folders').prop('checked'),
+                        'subtitles': $('#subtitles').prop('checked'),
+                        'anime': anime,
+                        'scene': $('#scene').prop('checked'),
+                        'default_status_after': $('#statusSelectAfter').val(),
+                    });
+                    return false;
```

Comment:
This was made redundant by using `e.preventDefault();` at the top of the function.